src/Components/SingleReview.jsx

```diff
@@ -29,11 +29,20 @@ function SingleReview (props) {
       });
     };
     
-  const updateVote = (increment) => {
-    setVoteCount((prevCount) => prevCount + (increment ? 1 : -1));
+  const updateVoteUp = () => {
+    setVoteCount((prevCount) => prevCount +  1 );
+    patchVotesIncrease(review_id).then((votesCount) => {
+      setVoteCount(votesCount)
+  })
   };
```

Comment:
You shouldnt update votes with the new value from the patch request if you are optimistically updating this value already.